unix/vncserver/vncserver.in

```diff
@@ -35,7 +35,17 @@
 # your site
 #
 
+# Start with legacy ~/.vnc user dir
 $vncUserDir = "$ENV{HOME}/.vnc";
+if (stat($vncUserDir)) {
+  warn "~/.vnc is deprecated, see https://github.com/TigerVNC/tigervnc/pull/1737";
```

Comment:
Reading the pull request can be very unclear. Especially for novice users. I'd prefer if we can have something more independent. Can we refer to something in one of the man pages instead?

Reply:
Yes, that occurred to me as soon as I started editing the man pages altogether. I'll fix that up as part of the documentation change.